dev-tools/mage/fields.go

```diff
@@ -80,6 +80,7 @@ func GenerateFieldsGo(fieldsYML, out string) error {
 		"-in", fieldsYML,
 		"-out", createDir(out),
 		"-license", toLibbeatLicenseName(BeatLicense),
+		"-name", "Beat",
```

Comment:
Not for myself: This should become a param to the function?

Reply:
Managed to remove this method completely, needs to be reintroduce later again differently.